sdk/digitaltwins/azure-digitaltwins-core/src/test/prerequisite/manifest.json

```diff
@@ -0,0 +1,9 @@
+[{
+    "resourceAppId": "0b07f429-9f4b-4714-9392-cc5e8e80c8b0",
```

Comment:
what are these values?

Reply:
this the the dataplane APP ID for ADT

Reply:
Why not use the human readable one: https://digitaltwins.azure.net?

Reply:
we no longer need this anyways, removing.